CommonCoreLegacy/build.gradle

```diff
@@ -18,6 +17,12 @@ android {
     testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
   }
 
+  testOptions {
+    unitTests {
+      includeAndroidResources = true
+    }
```

Comment:
Can u explain why we need these?

Reply:
Simple include the strings in the test resources. Mentioned in the docs http://robolectric.org/getting-started/